packages/contracts-bedrock/scripts/deploy/DeployImplementations.s.sol

```diff
@@ -230,8 +230,10 @@ contract DeployImplementations is Script {
         require(checkAddress == address(0), "OPCM-50");
         // The max initcode/runtimecode size is 48KB/24KB.
         // But for Blueprint, the initcode is stored as runtime code, that's why it's necessary to split into 2 parts.
-        (blueprints.permissionedDisputeGame1, blueprints.permissionedDisputeGame2) = DeployUtils.createDeterministicBlueprint(vm.getCode("PermissionedDisputeGame"), _salt);
-        (blueprints.permissionlessDisputeGame1, blueprints.permissionlessDisputeGame2) = DeployUtils.createDeterministicBlueprint(vm.getCode("FaultDisputeGame"), _salt);
+        if (!DevFeatures.isDevFeatureEnabled(_input.devFeatureBitmap, DevFeatures.DEPLOY_V2_DISPUTE_GAMES)) {
```

Comment:
I know we have the ChainAssertions, but still probably a good idea to verify the new behavior with a quick unit test.  I think we can just tweak [the existing test here](https://github.com/ethereum-optimism/optimism/blob/aa6238aafed9bffc690f284d219433c8044853bd/packages/contracts-bedrock/test/opcm/DeployImplementations.t.sol#L326-L326).

Reply:
Updated the unit test.